### PR TITLE
OSDOCS-11382: Renaming the user in ANP

### PR DESCRIPTION
**Pull request by mpandya15**
OSDOCS#11382: Renaming the user in ANP

Version(s):

Issue:
https://issues.redhat.com/browse/OSDOCS-11382

Link to docs preview: https://79396--ocpdocs-pr.netlify.app/

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/modifying-template-for-new-projects.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [id="modifying-template-for-new-projects_{context}"]
 = Modifying the template for new projects
 
-As a cluster administrator, you can modify the default project template so that
+As a developer, you can modify the default project template so that
```
Comment (asood-rh) [inline modules/modifying-template-for-new-projects.adoc:10]: I do not think this statement should be changed as this is about creating project template, not about network policies

File: modules/nw-networkpolicy-project-defaults.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [id="nw-networkpolicy-project-defaults_{context}"]
 = Adding network policies to the new project template
 
-As a cluster administrator, you can add network policies to the default template for new projects.
+As a developer, you can add network policies to the default template for new projects.
```
Comment (asood-rh) [inline modules/nw-networkpolicy-project-defaults.adoc:11]: Same as above.

File: networking/network_security/network_policy/about-network-policy.adoc [hunk 3]
```diff
@@ -9,7 +9,7 @@ endif::openshift-dedicated,openshift-rosa[]
 
 toc::[]
 
-As a cluster administrator, you can define network policies that restrict traffic to pods in your cluster.
+As a developer, you can define network policies that restrict traffic to pods in your cluster.
```
Comment (asood-rh) [inline networking/network_security/network_policy/about-network-policy.adoc:12]: This is the only statement that needs to be modified.

File: modules/modifying-template-for-new-projects.adoc (hunk 1 above)
Comment (JoeAldinger) [reply modules/modifying-template-for-new-projects.adoc:10]: Can only cluster administrators create new projects?

Comment (asood-rh) [reply modules/modifying-template-for-new-projects.adoc:10]: @JoeAldinger It is creating a project versus creating a project template. The project template once created by cluster admin can be used by developer.
https://docs.openshift.com/container-platform/4.16/applications/projects/configuring-project-creation.html

**Review by sheriff-rh — APPROVED** [review]
LGTM

**Comment by sheriff-rh** [comment]
@mpandya15 reminder to add which release versions this change applies to before you're ready to merge.

**Pull request merged**